core/src/main/resources/hudson/PluginManager/table_de.properties

```diff
@@ -33,8 +33,11 @@ Install\ without\ restart=Installieren ohne Neustart
 Download\ now\ and\ install\ after\ restart=Jetzt herunterladen und bei Neustart installieren
 compatWarning=\
   Warnung: Die neue Version ist nicht kompatibel mit der momentan installierten Version. \
-  Jobs, welche dieses Plugin verwenden, m�ssen gegebenenfalls neu konfiguriert werden.
+  Jobs, welche dieses Plugin verwenden, m\u00FCssen gegebenenfalls neu konfiguriert werden.
 coreWarning=\
-  Warnung: Dieses Plugin wurde f�r Jenkins {0} oder neuer entwickelt. Es kann mit Ihrer \
+  Warnung: Dieses Plugin wurde f\u00FCr Jenkins {0} oder neuer entwickelt. Es kann mit Ihrer \
   Jenkins-Installation funktionieren... oder auch nicht.
-Click\ this\ heading\ to\ sort\ by\ category=Hier klicken, um nach der Kategorie zu sortieren
\ No newline at end of file
+Click\ this\ heading\ to\ sort\ by\ category=Hier klicken, um nach der Kategorie zu sortieren# \ 
```

Comment:
This is the sort of madness caused by not having https://github.com/jenkinsci/jenkins/pull/2779